bodhi/tests/server/test_mail.py

```diff
@@ -287,7 +287,7 @@ def test_recipients_refused(self, SMTP, warn):
         SMTP.assert_called_once_with('smtp.fp.o')
         smtp.sendmail.assert_called_once_with('archer@spies.com', ['lana@spies.com'], 'hi')
         warn.assert_called_once_with(
-            '"recipient refused" for \'lana@spies.com\', SMTPRecipientsRefused(\'nooope!\',)')
+            '"recipient refused" for \'lana@spies.com\', SMTPRecipientsRefused(\'nooope!\'{})'.format(base._erepr_end))
```

Comment:
Bodhi runs flake8 on the code, and requires lines to be no longer than 100 columns:

```
f27	./bodhi/tests/server/test_mail.py:290:101: E501 line too long (119 > 100 characters)
f27	            '"recipient refused" for \'lana@spies.com\', SMTPRecipientsRefused(\'nooope!\'{})'.format(base._erepr_end))
```

Reply:
Oh. I try to keep lines < 80, but here it clearly wan't the case, so i didn't care. Nevertheless the new approach is shorter.